src/main/java/org/fundacionjala/coding/rfalconi/SortInnerContent.java

```diff
@@ -0,0 +1,33 @@
+package org.fundacionjala.coding.rfalconi;
+
+import java.util.Arrays;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * SortInnerContent
+ * You have to sort the inner content of every word of a string in descending order.
+ * The inner content is the content of a word without first and the last char.
+ * Some examples:
+ * "sort the inner content in descending order" -> "srot the inner ctonnet in dsnnieedcg oredr"
+ */
+
+public class SortInnerContent {
```

Comment:
Don't commit this kata

Reply:
Files related to SortInnerContent kata were removed